code/modules/admin/verbs/randomverbs.dm

```diff
@@ -547,7 +547,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	var/msg = "<span class='danger'>Admin [key_name_admin(usr)] healed / revived [ADMIN_LOOKUPFLW(M)]!</span>"
 	message_admins(msg)
 	admin_ticket_log(M, msg)
-	SSblackbox.record_feedback("tally", "admin_verb", 1, "Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```

Comment:
I'd recommend against changing this one as it can damage data tracking for any downstream that actually uses it.

Reply:
This was merged into tg just fine :S if this is still the case, I'm not against reverting it

Reply:
What francium said

Reply:
Reverted